packages/valory/skills/dynamic_nft_abci/ceramic_db.py

```diff
@@ -69,12 +70,14 @@ def create_user(self, user_data):
 
         fields = self.USER_FIELDS.union(user_data.keys())
 
-        new_user = {
-            field: user_data.get(
-                field, 0 if field in ("points", "current_period_points") else None
-            )
-            for field in fields
-        }
+        new_user = {}
+        for field in fields:
+            if field in ("points", "current_period_points"):
+                new_user[field] = user_data.get(field, 0)
+            elif field in ("tweet_id_to_points"):
```

Comment:
This is going be treated as a literal not tuple, but since it's string I don't think it's going to cause an issue

Reply:
I've just checked that it works.

Reply:
Just make sure this is what you want, because:
```
"test" is "test" -> True
"t" is "test" -> True  # Also True, but might not be what you want in this case. IMO this should go in else
```

Alternatively, you can convert this to:

```python
elif field == "tweet_id_to_points":
```

Reply:
I'll add a comma to be consistent with the other clauses.